### PR TITLE
Add Task

### PR DESCRIPTION
**Pull request by AbdelrhmanWalaa**
Add "RGB LED Control V1.0" Task

### CODE REVIEW

File: 10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_program.c [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	while (1)
+	{
+		/* Step 1: Get the BTN state. */
+		BTN_getBTNState( BTN_U8_3, &u8_l_btnValue );
+		
+		/* Check 1: BTNValue is Low. */
+		if ( u8_l_btnValue == BTN_U8_LOW )
+		{
+			/* Check 1.1: Required Counter. */
+			switch ( u8_l_counter )
```
Comment (Hala-Abdelkader) [inline 10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_program.c:59]: Switch case should have only one break

File: 10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_interface.h [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+/* LIB */
+#include "std_types.h"
+#include "bit_math.h"
+
+/* HAL */
+#include "led_interface.h"
+#include "btn_interface.h"
+
+/* MCAL */
+#include "delay_interface.h"
```
Comment (Hala-Abdelkader) [inline 10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_interface.h:24]: You shouldn't include drivers from MCAL layer in application

Comment (AbdelrhmanWalaa) [reply 10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_interface.h:24]: Thanks for your feedback🤲🏼, however I justified this in the design document.

File: 10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_program.c (hunk 1 above)
Comment (AbdelrhmanWalaa) [reply 10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_program.c:59]: Thanks for your feedback🤲🏼.
Task description did not obligate to comply with any of the MISRA standards.

**Pull request merged**